### PR TITLE
Fix bossaction walkover specials

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
https://github.com/kraflab/dsda-doom/commit/3bd7ead88a482ee0ea0a00cf00b4e1fa92ced3b0

**Review by rfomin — APPROVED** [review]

**Comment by fabiangreffrath** [comment]
We'll wait for @kraflab's approval here.

**Review by kraflab — APPROVED** [review]

**Pull request merged**
